rook/migration/v1.11.x-to-v1.14.x.md

```diff
@@ -0,0 +1,85 @@
+# Upgrade rook-ceph to v1.11.x to v1.14.x
+
+## Prerequisites
+
+- [ ] Check the state of rook-ceph so it is healthy before you begin:
+
+  ```bash
+  helm -n rook-ceph list --all
+  # should show all deployed
+  kubectl -n rook-ceph get jobs
+  # should show all completed
+  kubectl -n rook-ceph get pods
+  # should show all running or completed
+  kubectl -n rook-ceph get cephclusters
+  # should show PHASE = Ready and HEALTH = HEALTH_OK
+  ```
+
+## Apply upgrade 1.12.x - _disruptive_
+
+- 1. Fetch chart version:
+
+```console
+sudo rm -rf helmfile.d/upstream/rook-ceph
+helm fetch rook-release/rook-ceph --version v1.12.11 --untar --destination helmfile.d/upstream
+sudo rm -rf helmfile.d/upstream/rook-ceph-cluster
+helm fetch rook-release/rook-ceph-cluster --version v1.12.11 --untar --destination helmfile.d/upstream
+```
+
+- 2. Upgrade rook-ceph:
+
+```console
+helmfile -e workload apply
+```
+
+## Apply upgrade 1.13.x - _disruptive_
```

Comment:
I would consider adding the pre-checks between each version upgrade :slightly_smiling_face: